cogs/fixtures.py

```diff
@@ -91,7 +91,17 @@ async def next(self, ctx):
     async def results(self, ctx):
         fixtures = parseResults()
         body = findResults(fixtures)
-        await ctx.send(f"```{body}```")
+
+        embed = discord.Embed(
+            color=0x9C824A,
+            description=f"```{body}```")
+
+        embed.set_author(
+            name="Last 3  results",
```

Comment:
Double space here

Reply:
One of these days I will add Black to this repo 👀